templates/blog.jade

```diff
@@ -39,11 +39,20 @@ block main
 
       div.Grid.Grid--1of2.Grid--gutters-lg
         each article, i in articles.slice(1)
+
           - var img_src = (article.metadata.featured_img_thumb ? article.metadata.featured_img_thumb : "https://placehold.it/800x600")
 
             div.Grid-cell
-              div.Feature-image--blog-archive--wrapper
-                div(src=img_src style="background-image: url('"+img_src+"');").Background-img--blog-archive.Background-img-contain
+
+              if article.metadata.tag
```

Comment:
quick fix for working markup below - it would be nice to have less "gridception" here - see if this can be simplified and inline styles removed if possible

```jade
              if article.metadata.tag
                div.Feature-image--blog-archive--wrapper.Grid--center.Grid--justifyCenter.Grid-cell--fillHeight
                  div.Grid-cell.Grid-cell--full
                    div(style="flex-direction:column;").Grid.Grid--center.Grid--justifyCenter.Grid-cell--fillHeight
                    div.Grid-cell
                        div.Circle.Tag-inner
                          i.material-icons.Icon-tag= article.metadata.tag.icon
                          h5.Caption-tag= article.metadata.tag.caption
```